app/models/manageiq/providers/ansible_tower/automation_manager/machine_credential.rb

```diff
@@ -1,2 +1,66 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::MachineCredential < ManageIQ::Providers::AutomationManager::Authentication
+  FIELDS = {
+    :description => {
+      :type       => :string,
+      :required   => false,
+      :label      => _('Description'),
+      :help_text  => _('Optional description of this credential.'),
+      :encrypted  => false
+    },
+    :username         => {
+      :type       => :string,
+      :required   => false,
+      :label      => _('Username'),
+      :help_text  => _('Username for this credential.'),
+      :max_length => 1024,
+      :encrypted  => false
+    },
```

Comment:
@blomquisg @bdunne Thoughts?  When we discussed this with the UI team, I thought we agreed that the fields at the "top", like username, password, and description would just be regular columns, then the *rest* would be extra_attributes, with the password values being unfetchable